### ISSUE TITLE
How does Embedding layer works if input value greater than input_dim?

### ISSUE THREAD

**Issue by smirnovevgeny** [keras-team/keras#13989]
How does Embedding layer works if input value greater than input_dim?

Why keras doesn't raise an exception?

`from keras.models import Sequential`
`from keras.layers import Embedding
`
`model = Sequential()`
`model.add(Embedding(1, 2, trainable=True, mask_zero=False))`
`input_array = [5]
`
`model.compile("rmsprop", "mse")
`
`output_array = model.predict(input_array)
`
`outpur_array
`
`#array([[[0., 0.]]], dtype=float32)
`

The [keras.applications](https://github.com/keras-team/keras-applications) module has been splitted from the main Keras repository.

Please submit your issue using this [link](https://github.com/keras-team/keras-applications/issues/new).

Thank you!

**Comment by zabir-nabil** [comment]
It seems keras 2.3.1 just maps out of vocabulary word tokens to zero vector. Referencing your SO question here too for a later complete answer

[Stackoverflow question](https://stackoverflow.com/questions/61342481/how-does-keras-embedding-layer-works-if-input-value-greater-than-input-dim/61345720#61345720)

**Comment by smirnovevgeny** [comment]
@zabir-nabil You are right, but will keras fix the bag?

**Comment by zabir-nabil** [comment]
@smirnovevgeny unfortunately, it got a `support` label instead of a bug, also don't see anyone getting assigned to the issue, so I don't see getting it fixed anytime soon. You can try creating another issue with a proper bug format, (current behavior and expected behavior) with mention to your question and the answer which explains how the behavior is inconsistent with `tf.keras`, that may draw some attention.

**Comment by sushreebarsa** [comment]
@smirnovevgeny  Moving this issue to closed status as there has been no recent activity, in case you still face the error please create a new issue.Thanks!
